test/Conversion/FIRRTLToHW/lower-to-hw.mlir

```diff
@@ -495,12 +498,6 @@ firrtl.circuit "Simple"   attributes {annotations = [{class =
     // CHECK-NEXT:     sv.ifdef "USE_PROPERTY_AS_CONSTRAINT" {
     // CHECK-NEXT:       sv.assume.concurrent posedge %clock, [[TMP2]]
     // CHECK-NEXT:     }
-    // CHECK-NEXT:   }
-    // CHECK-NEXT: }
-    firrtl.assume %clock, %cond, %enable, "assume0" {isConcurrent = true, guards = ["HELLO", "WORLD"]}
-    firrtl.cover %clock, %cond, %enable, "cover0" {isConcurrent = true, guards = ["HELLO", "WORLD"]}
-    // CHECK-NEXT: sv.ifdef "HELLO" {
-    // CHECK-NEXT:   sv.ifdef "WORLD" {
```

Comment:
This part is not related to FIRRTL_BEFORE_INITIAL/FIRRTL_AFTER_INITIAL. Ifdefs are not merged because of invalid use of SmallDenseMap(I guess).